maintenance/toggleExtension.php

```diff
@@ -9,30 +9,66 @@
 require_once "$IP/maintenance/Maintenance.php";
 
 use Maintenance;
-use MediaWiki\MediaWikiServices;
+use MediaWiki\WikiMap\WikiMap;
 use Miraheze\ManageWiki\Helpers\ManageWikiExtensions;
 
 class ToggleExtension extends Maintenance {
 	public function __construct() {
 		parent::__construct();
 		$this->addArg( 'ext', 'The ManageWiki name of the extension.', true );
 		$this->addOption( 'disable', 'Disable the extension. If not given, enabling is assumed.' );
+		$this->addOption( 'all-wikis', 'Run on all wikis present in $wgLocalDatabases.' );
+		$this->addOption( 'no-list', 'Only list wikis after it has ran on them. This may speed up execution.' );
```

Comment:
Whoops, I guess this is kinda an inaccurate description.

Reply:
```suggestion
		$this->addOption( 'quiet', 'Suppress output' );
```

could be a better name and description.

Reply:
I thought about that but it doesn't really suppress output just doesn't list each one, it still has output at the end.

Reply:
Hmm, how about this then?

```suggestion
		$this->addOption( 'no-list', "Don't list on which wikis this script has ran." );
```

Reply:
actually, not even I like that last suggestion.